src/test/java/spoon/test/refactoring/ChangeVariableNameTest.java

```diff
@@ -0,0 +1,244 @@
+package spoon.test.refactoring;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import spoon.Launcher;
+import spoon.OutputType;
+import spoon.SpoonException;
+import spoon.SpoonModelBuilder;
+import spoon.refactoring.ChangeLocalVariableName;
+import spoon.reflect.code.CtLocalVariable;
+import spoon.reflect.declaration.CtAnnotation;
+import spoon.reflect.declaration.CtClass;
+import spoon.reflect.declaration.CtElement;
+import spoon.reflect.declaration.CtMethod;
+import spoon.reflect.declaration.CtType;
+import spoon.reflect.declaration.CtVariable;
+import spoon.reflect.factory.Factory;
+import spoon.reflect.reference.CtTypeReference;
+import spoon.test.refactoring.testclasses.TryRename;
+import spoon.test.refactoring.testclasses.VariableRename;
+import spoon.testing.utils.ModelUtils;
+
+public class ChangeVariableNameTest
+{
+	Launcher launcher;
+	Factory factory;
+	CtClass<VariableRename> varRenameClass;
+	CtTypeReference<TryRename> tryRename;
+	CtLocalVariable<?> local1Var;
+
+	@Before
+	public void setup() throws Exception {
+		varRenameClass = (CtClass<VariableRename>)ModelUtils.buildClass(VariableRename.class);
+		launcher = new Launcher();
+		launcher.addInputResource("./src/test/java/spoon/test/refactoring/testclasses/VariableRename.java");
+		File outputBinDirectory = new File("./target/spooned-refactoring-test");
+		if (!outputBinDirectory.exists()) {
+			outputBinDirectory.mkdirs();
+		}
+		launcher.setBinaryOutputDirectory(outputBinDirectory);
+		launcher.setSourceOutputDirectory(outputBinDirectory);
+		launcher.buildModel();
+		factory = launcher.getFactory();
+		varRenameClass = factory.Class().get(VariableRename.class);
+		tryRename = factory.createCtTypeReference(TryRename.class);
+		local1Var = varRenameClass.filterChildren((CtLocalVariable<?> var)->var.getSimpleName().equals("local1")).first();
+	}
+
+	@Test
+	public void testModelConsistency() throws Throwable {
+		new VariableRename();

```

Comment:
It has no side effects, but it throws an exception if the tested model is inconsistent. It is similar to `VariableReferencesModelTest`, where I first tested model consistency by call of constructor, which internally called all other methods of model to check if they contain consistent assertions. Then you came with good idea to switch it to jUnit test class too. So `VariableRename` is very similar to that, but in this case I cannot easily switch it to jUnit test class, because I am testing model consistency before and after each model refactoring test action.